app/views/articles/_posts-form.html.erb

```diff
@@ -0,0 +1,25 @@
+<ul class='posts-index-posts'>
+    <% @posts.each do |post| %>
+      <li class='posts-index-post'>
+        <section class='posts-index-post-horizontal'>
+          <p class='posts-index-post-user-icon'>
+            <%= image_tag("user_icon.png", :alt => "user icon") %>
+          </p>
+          <p class="posts-index-post-user-name">
+            User Name
+          </p>
+        </section>
+        <p class='posts-index-post-message'>
+          <%= post.message %>
+        </p>
+        <p class="posts-index-post-timestamp">
+          <%= post.created_at.localtime.strftime('%d-%b-%Y %I:%M:%S %p') %>
+        </p>
+        <p><%= link_to 'Comments', post_path(post) %></p>
+      </li>
+    <% end %>
+</ul>
+
+<%= link_to new_post_path do %>
+  New post
+<% end %>
```

Comment:
We made the form a partial as it is being used in two different erb files now ('/' & /posts'), we placed the partial in a folder called 'Articles'. Article is a naming convention that we followed from the Rails docs.